src/localstorage.ts

```diff
@@ -66,8 +65,9 @@ export default class LocalStorage extends CachingLayer {
 
 
   // TODO fix this
-  _emit (...args: any[]): void {
-    return;
+  // NOTE: will be overwritten by eventHandlingMíxin
+  _emit(...args: any[]): void {
```

Comment:
Same as above: not sure if necessary. But if it is, then it should return as `never`.